dask_jobqueue/core.py

```diff
@@ -322,6 +327,7 @@ def _calls(self, cmds):
         for cmd in cmds:
             logger.debug(' '.join(cmd))
             procs.append(subprocess.Popen(cmd,
+                                          shell=self.popen_shell,
```

Comment:
I recommend that we pass `**kwargs`, or at least `shell=` from the `_call` method into this function call and avoid the state entirely.

```python
def _call(self, cmd, **kwargs):
    return self._calls([cmd], **kwargs)

def _calls(self, cmds, **kwargs):
    ...
    procs.append(subprocess.Popen(cmd,
                                  stdout=subprocess.PIPE,
                                  stderr=subprocess.PIPE,
                                  **kwargs))
```

Then we can call this like `self._call(cmd, shell=True)` and avoid mucking about with the `popen_shell` state.